lib/rubocop/cop/rspec/predicate_matcher.rb

```diff
@@ -106,7 +106,7 @@ def rewrite_matcher(corrector, predicate, matcher)
 
         def true?(to_symbol, matcher)
           result = case matcher.method_name
-                   when :be, :eq
+                   when :be, :eq, :eql, :equal
```

Comment:
A few aliases to make this complete? https://github.com/rspec/rspec/blob/a80119c703d38f993c414490abe8c992154cf5d1/rspec-expectations/lib/rspec/matchers.rb#L578

I just went there just in case, I don't think I've seen those used once in my practice 🙈  So really up to you.

Reply:
Yeah, I have never seen those used either. I think there’s no need to add them now.